test/CMakeLists.txt

```diff
@@ -42,10 +52,13 @@ add_definitions(-DEASTL_THREAD_SUPPORT_AVAILABLE=0)
 #-------------------------------------------------------------------------------------------
 add_executable(EASTLTest ${SOURCES})
 
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads REQUIRED)
+
 set(EASTLTest_Libraries
     EASTL
     EATest)
-target_link_libraries(EASTLTest ${EASTLTest_Libraries})
+target_link_libraries(EASTLTest ${EASTLTest_Libraries} Threads::Threads)
```

Comment:
std::this_thread::sleep_for in eathread.h calls nanosleep internally. nanosleep is provided by pthreads.